optuna/distributions.py

```diff
@@ -81,27 +73,23 @@ def _contains(self, param_value_in_internal_repr):
 
         raise NotImplementedError
 
-    def _asdict(self):
-        # type: () -> Dict
+    def _asdict(self) -> Dict:
```

Comment:
Can we add types of key and value like L336?
```suggestion
    def _asdict(self) -> Dict[str, Any]:
```

Reply:
Let me decline this proposal as @hvy said in https://github.com/optuna/optuna/pull/1513#pullrequestreview-448738260.